tests/test_operators.py

```diff
@@ -45,23 +46,86 @@ def compare_streams(self, all, expected_all):
             self.assertDictEqual(input_dict, output_dict)
 
     def test_map_instance_values(self):
+        mappers = {"a": {"1": "hi", "2": "bye"}}
+
         inputs = [
-            {"a": 1, "b": 2},
-            {"a": 2, "b": 3},
+            {"a": "1", "b": "2"},
```

Comment:
I thinking keeping the input  as numbers is a more realistic example.

Reply:
Hi @yoavkatz ,
We have a hidden problem here: the mappers are defined as  str->str dictionaries, and when processing, the input values are first transformed to strings, then looked up at the dictionary, and the output is str again. Not back to int.  So any mapped value is always a str.  Could this make a problem?

Reply:
I think the use of string internally is fine.  but people use it to map inputs that are not string, so we should keep this example.

Usually from int to string - as above.